### DISCUSSION TITLE
Compile RD client on Windows 10/Ubuntu 20

### DISCUSSION THREAD

**Discussion by SlalomJohn** [rustdesk/rustdesk#4526] [General]
On Windows 10 x64 -  **--release** compiled, but binary not work - got error
![image](https://github.com/rustdesk/rustdesk/assets/65710644/133f10b7-6621-458f-9520-4adf9725b23c)

On Ubuntu 20 LTS - compiler got error:
cargo b --target=x86_64-pc-windows-gnu --release
![image](https://github.com/rustdesk/rustdesk/assets/65710644/fb61e6ff-641a-459e-b3bc-bfd932e729a9)

cargo b --target=x86_64-pc-windows-msvc --release
![image](https://github.com/rustdesk/rustdesk/assets/65710644/ed0f428b-5f2e-4cca-9520-bb8f59629756)

why?

**Comment by SlalomJohn** [comment]
Nobody has an answer?
For everyone - you were able to compile from source the client for Windows ???
In lunux or windows compiler, docker or directly ???

**Comment by SergeyMy** [comment]
1. https://github.com/rustdesk/rustdesk/releases/tag/nightly
2. CROSS_COMPILE assembly quest, the main rake with LLVM and clang

**Reply by SlalomJohn** [reply]
> 1. https://github.com/rustdesk/rustdesk/releases/tag/nightly
> 2. CROSS_COMPILE assembly quest, the main rake with LLVM and clang

not compile
![image](https://github.com/rustdesk/rustdesk/assets/65710644/24918881-2542-4890-97db-03b6e3b4ec07)

**Reply by SlalomJohn** [reply]
OMG... All user may be compile CLIENT binary for Windows?

**Reply by SergeyMy** [reply]
I don’t know about windows, I only have linux :)

**Reply by SlalomJohn** [reply]
Server-side on linux is compiled sucess and work.
But client-side on Windows - can't compile....
